calculator/src/main/java/com/market/banica/calculator/service/CalculatorServiceImpl.java

```diff
@@ -19,6 +25,7 @@
 public class CalculatorServiceImpl implements CalculatorService {
 
     AuroraClientSideService auroraService;
```

Comment:
please make the fields 'private final'

Reply:
The calculator service class code is added for the sole purpose of integration tests, it connects the rest services with data from the order book service, it is now being reworked in the new branch for food item price optimization logic. It has no intention to represent real use case scenario but to mock the behaviour of the real class to be implemented.